test/e2e/cli_test.go

```diff
@@ -822,6 +822,16 @@ func (s *CLISuite) TestCron() {
 		})
 	})
 
+	s.Run("Create Parameter Override", func() {
+		s.Given().RunCli([]string{"cron", "create", "cron/param.yaml", "-p", "message=\"bar test passed\""}, func(t *testing.T, output string, err error) {
+			assert.NoError(t, err)
+		}).RunKubectl([]string{"get", "cwf", "test-cron-wf-basic-param", "-o", "yaml"}, func(t *testing.T, output string, err error) {
```

Comment:
why do we need to use `kubectl`  here?

Reply:
An easy way to verify that the parameter override succeeded is to check the resulting CronWorkflow resource. `argo cron get <NAME>` does not return the relevant resource contents

Reply:
Refactored this to no longer need `kubectl`